tools/XmlDocToMarkdown/XmlDocToMarkdown/MarkDownLibrary.cs

```diff
@@ -103,7 +104,8 @@ private static StringBuilder GenerateMarkDownForConstructors(Type t, IEnumerable
                methodName + "(" + string.Join(",", methodParams.Select(pi => pi.ParameterType.FullName)) + ")" :
                methodName;
 
-                var current = GetMarkdownForMethod(members, t.FullName + fullMethodName);
+                var fullName = t.FullName.Replace("+", ".");
```

Comment:
this is another special case where the method name has "+" (when you call assembly.gettype()).